infra/wca_on_rails/staging/auxiliary_services.tf

```diff
@@ -104,6 +104,11 @@ resource "aws_ecs_service" "auxiliary" {
 
   enable_execute_command = true
 
+  deployment_circuit_breaker {
+    enable   = true
+    rollback = false
+  }
```

Comment:
What does this do? I cannot infer it from the name alone

Reply:
This is good to have in every ECS Service.
It stops non Blue Green deployments after they failed 3 times. I changed this in the console for the staging auxiliary service after I saw that it failed deployment 600 times due to the staging redis being half full. It's not technically part of this PR, but the terraform files need to always match the state of what's on AWS or they will revert it.